README.md

```diff
@@ -103,6 +103,19 @@ Hutch::Logging.logger = Rails.logger
 See this [RabbitMQ tutorial on topic exchanges](http://www.rabbitmq.com/tutorials/tutorial-five-ruby.html)
 to learn more.
 
+### Message Processing Tracers
+
+Tracers allow you to track message processing. You can set them using `tracer` method.
+
+#### NewRelic
+```ruby
+class FailedPaymentConsumer
+  include Hutch::Consumer
+  consume 'gc.ps.payment.failed'
+  tracer 'NewRelic'
```

Comment:
Does your note about passing a class (instead of this Rails-y DSL) stand still in the config case? If so, we would need to "constantize" the `tracer` value from the config as we get it as a string. Or should we just explicitly warn in the README that this option can be set only using Hutch::Config#set?